controllers/administration.js

```diff
@@ -1515,6 +1515,7 @@ router.get(
 					schoolUsesLdap: res.locals.currentSchoolData.ldapSchoolIdentifier,
 					referrer: req.header('Referer'),
 					CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS,
+					hasAccount: !!account,
```

Comment:
user_edit.hbs is used to edit all users not only students.
Please check the teacher route, otherwise this property will always be falsy for teachers.

Reply:
Thanks @pkleybolte for the tip, I've added the validation on the teachers route as well.